Doppler.BigQueryMicroservice/Repository/Classes/UserAccessByUserRepository.cs

```diff
@@ -73,6 +76,41 @@ public async Task<IReadOnlyList<UserAccessByUser>> GetAllByUserIdAsync(string ac
                 }
             }
         }
+
+        public async Task<bool> MergeEmailsAsync(string accountName, List<string> emails)
+        {
+            using (var connection = await base.CreateConnectionAsync())
+            {
+                try
+                {
+                    var dt = new DataTable("[dbo].[TypeEmail]");
+                    dt.Columns.Add("Email", typeof(string));
+
+                    foreach (var email in emails)
+                    {
+                        dt.Rows.Add(email);
+                    }
+
+                    var result = await connection.QueryAsync("EXEC [datastudio].SaveEmailsByAccounName @P1_Emails, @P2_AccounName", new
```

Comment:
just a little typo: SaveEmailsByAccountName and the paramter @P2_AccountName

Reply:
ok suggest accepted and applied